features/page-objects/WorkspacePage.js

```diff
@@ -10,18 +10,23 @@ class WorkspacePage extends Page {
     this.driver.get(`${this.baseUrl}/workspaces/${workspace.slug}`);
   }
 
-  enterRoom(room) {
-    const roomCard = this.findRoomCard(room);
+  enterRoomThruUrl(workspace, room) {
+    this.driver.get(`${this.baseUrl}/workspaces/${workspace.slug}/rooms/${room.slug}`);
+  }
+
+  async enterRoom(room) {
+    const roomCard = await this.findRoomCard(room);
     roomCard.findElement(By.linkText("Enter Room")).click();
   }
 
-  findRoomCard(room) {
-    return this.driver.findElement(By.id(room.name))
+  async findRoomCard(room) {
+    await this.driver.wait(until.elementLocated(By.id(room.name)));
+    return await this.driver.findElement(By.id(room.name));
   }
 
   async videoPanel() {
-    await this.driver.wait(until.elementLocated(By.css("[name='jitsiConferenceFrame0']")));
-    return await this.driver.findElement(By.css("[name='jitsiConferenceFrame0']"));
+    await this.driver.wait(until.elementLocated(By.css("[name*='jitsiConferenceFrame']")));
+    return await this.driver.findElement(By.css("[name*='jitsiConferenceFrame']"));
```

Comment:
Relax the selector a bit because when entering a second room, we will have jitsiConferenceFrame1 instead.

Reply:
Hmm, that's concerning. Does that mean we're inserting a new iFrame every time we switch rooms? Is there a possibility for leaking Video/Audio to a room unintentionally then?

Reply:
Legit concern, will fix in the next pr.